turbopack/crates/turbopack-node/src/evaluate.rs

```diff
@@ -525,19 +525,24 @@ async fn pull_operation<T: EvaluateContext>(
     let output = loop {
         match operation.recv().await? {
             EvalJavaScriptIncomingMessage::Error(error) => {
+                let guard = duration_span!("error");
```

Comment:
Should we keep those spans? We do generally have to ensure to not add tracing everywhere to still be able to view the traces of giant apps (which are already ~8gb in size).

Reply:
yeah we can drop these, they didn't end up being useful anyway.